lib/autoproj/cli/main_test.rb

```diff
@@ -22,6 +22,21 @@ def report(report_options = Hash.new)
                 end
             end
 
+            desc 'default [on|off]', 'set whether tests are enabled or disabled by default, without touching existing settings'
+            def default(on_or_off)
```

Comment:
How about `autoproj enable --default` and `autoproj disable --default` ?

Reply:
I thought about it, but liked the explicit `autoproj default`. The reason was that both commands have fundamentally different signatures:

- `autoproj test enable|disable [PACKAGES]`
- `autoproj test default (on|off)`